x-pack/plugins/infra/public/components/document_title.tsx

```diff
@@ -47,19 +47,19 @@ const wrapWithSharedState = () => {
       return null;
     }
 
-    private getTitle(title: TitleProp) {
+    public getTitle(title: TitleProp) {
```

Comment:
This is returning an anonymous class expression so `private`/`protected` are not permitted if we're emitting declarations. (TS4094/Microsoft/TypeScript#30355)